.eslintignore

```diff
@@ -2,6 +2,7 @@ node_modules
 **/.next/**
 **/_next/**
 **/dist/**
+e2e-tests/**
```

Comment:
Don't see this dir used anywhere

```suggestion
```

Reply:
It's created here https://github.com/vercel/next.js/blob/eecc3342cb417ed22341985afb566d1e128cacdd/test-pnp.sh#L28 and gitignored here https://github.com/vercel/next.js/blob/ae77be53f7f57c0f22ee7276347e3a30b680d998/.gitignore#L24